packages/next/next-server/server/config.ts

```diff
@@ -229,6 +229,14 @@ export default function loadConfig(
       phase,
       userConfigModule.default || userConfigModule
     )
+
+    if (Object.keys(userConfig).length === 0) {
+      console.warn(
+        chalk.yellow.bold('Warning: ') +
```

Comment:
@Timer is this consistent with other warnings we show?

Reply:
Yup!